.gitignore

```diff
@@ -67,3 +67,6 @@ electron-builder.win-store.yaml
 
 # ignore changes
 ./src/environments/versions.ts
+
+#Notes
+notes.txt
```

Comment:
that's not necessary i think. can you please resubmit this pr without the change?